new-dti-website/components/initiatives/InitiativeDisplay.tsx

```diff
@@ -49,7 +46,7 @@ const Initiative = ({
           width >= LAPTOP_BREAKPOINT ? inter.className : ibm_plex_mono.className
         }`}
       >
-        {parts.map((part, index) => (part === eventName ? <b key={index}>{part}</b> : part))}
+{description}
       </p>
     </article>
   );
```

Comment:
FYI `eventName` still appears below too (line 68 and 70), and type checker caught it

Reply:
ohh good catch, should be fixed now i think...